index.html

```diff
@@ -64,15 +64,16 @@ <h2>Search for a Joke</h2>
                 <option value="sport">sport</option>
                 <option value="travel">travel</option>
               </select>
+              <button class="search-button" id="generate">Get Joke</button>
             </div>
-            <button class="search-button" id="generate">Get Joke</button>
           </div>
+          <div class="middle_separator"></div>
           <div class="search-right">
+            <label for="keyword">By Keyword</label>
             <div class="search-keyword" id="by-keyword">
-              <label for="keyword">Keyword: </label>
```

Comment:
You need to still have a label for accessibility. This provides focus for a user without a mouse and works with screen readers. You can have this visibility: hidden. The reader will still say the label.

Reply:
@GillianTrethewey 
The labels are still there :)
I just moved them around. The Keyword label, for example is two lines above where it was originally.